pkg/operator/util/util.go

```diff
@@ -0,0 +1,34 @@
+package util
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	"github.com/rook/rook/pkg/operator/api"
+	rookclient "github.com/rook/rook/pkg/rook/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-util")
+var clientTimeout = 15 * time.Second
+
+// GetRookClient gets a reference of the rook client connected to the Rook-API on the given namespace
+func GetRookClient(namespace string, client kubernetes.Interface) (rookclient.RookRestClient, error) {
```

Comment:
two thoughts motivating the question... 1) This is getting the REST client. The api package manages the REST service, so it seems like a natural fit. 2) We have a preference in rook to find specific packages before putting things in a generic `util` package. 
On the other hand, we are planning to remove the REST api once we have all the CRDs in place, so this will be a moot point soon.